app/views/layout.erb

```diff
@@ -15,12 +15,18 @@
 <body>
   <div class="container">
     <strong>Together, we can TODO it!</strong>
+    <% if session[:user_id].nil? %>
```

Comment:
Having a `current_user` helper would be an improvement on looking directly at the session here.